prometheus/prometheus.rules.yml

```diff
@@ -36,3 +36,83 @@ groups:
+    for: 120s
+    labels:
+      severity: "3"
+    annotations:
+      description: '{{ $labels.instance }} has CPU utilization over 99% for more than 120s'
+      summary: Instance {{ $labels.instance }} high CPU utilization
+  - alert: ReadLatency
+    expr: sum(rate(scylla_storage_proxy_coordinator_read_latency_sum[30s])) by (instance)/(sum(rate(scylla_storage_proxy_coordinator_read_latency_count[30s])) by (instance) +1)>500
+    for: 60s
+    labels:
+      severity: "3"
+    annotations:
+      description: '{{ $labels.instance }} has average read latency over 500ms'
+      summary: Instance {{ $labels.instance }} high read average latency
+  - alert: Read95Latency
+    expr: histogram_quantile(0.95, sum(rate(scylla_storage_proxy_coordinator_read_latency_bucket[5m])) by (le, instance))>500
+    for: 60s
+    labels:
+      severity: "2"
+    annotations:
+      description: '{{ $labels.instance }} has 95th read latency over 500ms'
+      summary: Instance {{ $labels.instance }} 95th read high latency
+  - alert: InsertLatency
+    expr: sum(rate(scylla_storage_proxy_coordinator_write_latency_sum[30s])) by (instance)/(sum(rate(scylla_storage_proxy_coordinator_write_latency_count[30s])) by (instance) +1)>500
+    for: 60s
+    labels:
+      severity: "3"
+    annotations:
+      description: '{{ $labels.instance }} has average write latency over 500ms'
+      summary: Instance {{ $labels.instance }} high average write latency
+  - alert: Insert95Latency
+    expr: histogram_quantile(0.95, sum(rate(scylla_storage_proxy_coordinator_write_latency_bucket[5m])) by (le, instance))>500
+    for: 60s
+    labels:
+      severity: "2"
+    annotations:
+      description: '{{ $labels.instance }} has 95th write latency over 500ms'
+      summary: Instance {{ $labels.instance }} 95th write high latency
+  - alert: OpenConnections
+    expr: node_sockstat_sockets_used>5000
+    for: 60s
+    labels:
+      severity: "2"
+    annotations:
+      description: '{{ $labels.instance }} has over 5000 open connections'
+      summary: Instance {{ $labels.instance }} has many open connections
+  - alert: PendingCompactions
+    expr: scylla_compaction_manager_compactions>100
+    for: 60s
+    labels:
+      severity: "2"
+    annotations:
+      description: '{{ $labels.instance }} has more than 100 pending compactions'
+      summary: Instance {{ $labels.instance }} has many pending compactions
+  - alert: LowMemory
+    expr: node_memory_MemFree_bytes<500000000
+    for: 60s
+    labels:
+      severity: "2"
+    annotations:
+      description: '{{ $labels.instance }} has lower than 0.5 GB for the last 60s'
+      summary: Instance {{ $labels.instance }} with low free memory
+  - alert: HighCQLTransaction
+    expr: sum(rate(scylla_transport_requests_served[60s])) by (instance)>1000000
+    for: 60s
+    labels:
+      severity: "2"
+    annotations:
+      description: '{{ $labels.instance }} has more than 1 million CQL operations per seconds'
```

Comment:
Why is this a problem? Scylla working to fast?
I think this alert is not useful.